src/TuxMutableRenderMixin.js

```diff
@@ -0,0 +1,133 @@
+'use strict';
+
+var invariant = require('./TuxInvariant');
+
+// findPath FUNCTION: recursive function to discover the first path to the provided @param mutableTrait from deepSearch function
+// @param mutableTrait STRING trait to find the path of keys to within the searchObject
+// @param searchObject OBJECT:
+  // expected keys ANY: this object can contain any keys, but is expected to contain the key specified in mutableTrait
+// @param currentPath ARRAY: array of strings that lead to the specified mutableTrait within searchObject
+// @param mutablePath ARRAY: empty array that the valid path will be added to once found
+var findPath = function (mutableTrait, searchObject, currentPath, mutablePath) {
+  // if the searchObject is undefined or null, return as the path isn't down this branch
+  if (searchObject === undefined || searchObject === null) {
+    return;
```

Comment:
Should return `mutablePath` here rather than `undefined`.  Otherwise we will throw an error on line 29.